### PR TITLE
LOK-2518 Resolve reduction key from NodeInfo

### PR DESCRIPTION
**Pull request by Mohammad-Junid**
## Description
We need to resolve reduction key from NodeInfo.

For ex: reduction-key="%uei%:%dpname%:%nodeid%:%interface%:%parm[#2]%"  should be resolved by replacing all the tags here with actual values.

## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2518

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: alert/src/main/resources/db/changelog/hs-0.1.0/data/event_definition_data.csv [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-7,uei.opennms.org/internal/node/serviceUnreachable,Device Unreachable,%s:%s:%d,,INTERNAL
-8,uei.opennms.org/internal/node/serviceRestored,Device Service Restored,%s:%s:%d,%s:uei.opennms.org/internal/node/serviceUnreachable:%d,INTERNAL
+1,uei.opennms.org/generic/traps/SNMP_Cold_Start,SNMP Cold Start,%uei%:%dpname%:%nodeid%:%interface%,,SNMP_TRAP
+2,uei.opennms.org/generic/traps/SNMP_Warm_Start,SNMP Warm Start,%uei%:%dpname%:%nodeid%:%interface%,,SNMP_TRAP
+3,uei.opennms.org/generic/traps/SNMP_Authen_Failure,SNMP Authen Failure,%uei%:%dpname%:%nodeid%:%interface%,,SNMP_TRAP
+4,uei.opennms.org/generic/traps/SNMP_Link_Down,SNMP Link Down,uei.opennms.org/nodes/snmp/interfaceOperDown:%dpname%:%nodeid%:%parm[#1]%,,SNMP_TRAP
+5,uei.opennms.org/generic/traps/SNMP_Link_Up,SNMP Link Up,%uei%:%dpname%:%nodeid%:%parm[#1]%,uei.opennms.org/nodes/snmp/interfaceOperDown:%dpname%:%nodeid%:%parm[#1]%,SNMP_TRAP
+6,uei.opennms.org/generic/traps/SNMP_EGP_Down,SNMP EGP Down,%uei%:%dpname%:%nodeid%:%interface%,,SNMP_TRAP
+7,uei.opennms.org/internal/node/serviceUnreachable,Device Unreachable,%uei%:%dpname%:%nodeid%:%interface%,,INTERNAL
```
Comment (cgorantla) [inline alert/src/main/resources/db/changelog/hs-0.1.0/data/event_definition_data.csv:8]: for these i think we also need `%uei%:%dpname%:%nodeid%:%interface%service%` at the end

**Review by cgorantla — COMMENTED** [review]
Looks good, need small update.

File: alert/src/main/resources/db/changelog/hs-0.1.0/data/event_definition_data.csv (hunk 1 above)
Comment (Mohammad-Junid) [reply alert/src/main/resources/db/changelog/hs-0.1.0/data/event_definition_data.csv]: Done

**Review by cgorantla — APPROVED** [review]
LGTM!

**Pull request merged**